### ISSUE TITLE
'gum file' option shown in docs doesnt exist in the cli

### ISSUE THREAD

**Issue by cloudymax** [charmbracelet/gum#190]
**Describe the bug**
Docs describe a 'file' option, this option is not recognized by the cli

**To Reproduce**
run `gum file .`

```bash
gum: error: unexpected argument file, did you mean "filter"?
  
  │ User-Data Template: Usage: gum 
  
  A tool for glamorous shell scripts.
  
  Flags:
  -h, --help       Show context-sensitive help.
  -v, --version    Print the version number
  
  Commands:
  choose     Choose an option from a list of choices
  confirm    Ask a user to confirm an action
  filter     Filter items from a list
  format     Format a string using a template
  input      Prompt for some input
  join       Join text vertically or horizontally
  spin       Display spinner while running a command
  style      Apply coloring, borders, spacing to text
  write      Prompt for long-form text
  
  Run "gum  --help" for more information on a command.
```

**Expected behavior**
<img width="502" alt="Screenshot 2022-10-12 at 09 23 32" src="https://user-images.githubusercontent.com/84841307/195276849-a2733486-337e-4ec9-a10d-31877691a810.png">

**Screenshots**
<img width="502" alt="Screenshot 2022-10-12 at 09 22 36" src="https://user-images.githubusercontent.com/84841307/195276885-da3cb1a2-0403-497e-b797-a5b6bb34fff4.png">

**Desktop (please complete the following information):**
 - Ubuntu 22.04 Server

**Comment by keevan** [comment]
Hi @cloudymax, you might be running an older version of gum.

This should be available on v0.8.0 and you may need to update. I was able to run the command successfully on 0.8.0. Are you able to confirm the version of gum you have and possibly your method of install?

**Comment by cloudymax** [comment]
Thank you for the reply, I was indeed on an older version somehow. Everything is working now that I'm on 0.8.0 👍
